src/docs/asciidoc/reference/architecture/code-structure.adoc

```diff
@@ -1,2 +1,150 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
+
+The purpose of this section is to explain the various elements and the interaction between them of a *ZetaPush application*.
+
+In the first instance we will see the <<reference.adoc#architecture_organization_application, organization of a ZetaPush application>>, then we will see the <<reference.adoc#architecture_files_tree, role of each file or folder>>.
+
+[#architecture_organization_application]
+== Organization of an application
+
+A *ZetaPush application* includes several components.
+You can see this with the following picture that represents the files tree of a basic application:
+
+image::{images-dir}architecture/files-tree.png[align="center"]
+
+
+The `worker` folder contains your business logic (i.e. your back code) and the `front` folder contains your front code.
```

Comment:
"(i.e. your back code)" I don't think we should say that. Maybe we should say something like: "this is the code that is not directly visible to your users. It is often named as backend part"
"the `front` folder contains your front code." I think we can't explain front using front word. We should say something like: "this is the visible part of your application. This folder contains the code of the web pages displayed to your users. It is often named the frontend part"